ert3/engine/_export.py

```diff
@@ -49,13 +49,10 @@ def export(workspace_root: Path, experiment_name: str) -> None:
             workspace=workspace_root, experiment_name=experiment_name
         )
     )
-    response_names = (
-        set(
-            ert3.storage.get_ensemble_record_names(
-                workspace=workspace_root, experiment_name=experiment_name
-            )
+    response_names = set(
```

Comment:
I don't think you need to convert it to a set here. The motivation originally was to compute the difference between the two sets... Now, any iterable should do 🤷🏻